### DISCUSSION TITLE
Images with external link are saved to root folder

### DISCUSSION THREAD

**Discussion by kushidou** [colinwilson/lotusdocs#88] [Q&A]
I added images to markdown content like this: 

`![xxxxx](https://s2.loli.net/2023/09/19/wXdP2uqgTC1GtMa.png)`

but this image has been saved to the root folder, and, after building,  they look like this in html file:

`<img src="/wXdP2uqgTC1GtMa_6857319088645747168.png" alt="xxxxx" width="651" height="312" loading="lazy">`

I believe I have not configered any configuration about image in hugo.toml, and other images have the same problem. 

My problem is contrary to #55  . My server is too slow to provide images so I hope these images will be loaded from other cloud server.

**Comment by colinwilson** [comment]
This behaviour is due to [render-image.html](https://github.com/colinwilson/lotusdocs/blob/release/layouts/docs/_markup/render-image.html) markup render hook.

> My server is too slow to provide images

I understand. I will add an option to the config / render hook to accommodate your use case. I'll update here once it's completed. 👍
